src/node_report.cc

```diff
@@ -294,6 +294,10 @@ static void WriteNodeReport(Isolate* isolate,
         static_cast<bool>(uv_loop_alive(env->event_loop())));
     writer.json_keyvalue("address",
         ValueToHexString(reinterpret_cast<int64_t>(env->event_loop())));
+
+    // Report Event loop idle time
+    uint64_t idle_time = uv_metrics_idle_time(env->event_loop());
+    writer.json_keyvalue("loopIdleTimeSeconds", 1.0 * idle_time / 1e9);
```

Comment:
Is `idle_time` in nanoseconds? I thought @trevnorris said it was milliseconds in https://github.com/nodejs/node/pull/35940#issuecomment-725051411.

Reply:
The libuv code uses `uv_hrtime()` without any additional scaling, so, yes, nanoseconds.

Reply:
https://github.com/nodejs/node/blob/589b2a1244da2f04843652bda97ef55397cdf5b7/src/node_perf.cc#L446-L447 

this may make it very clear - the `nanoseconds` that comes is converted to `millis` if accessed through API.

Reply:
That makes sense. I was just confused by https://github.com/nodejs/node/pull/35940#issuecomment-725051411. Thanks Anna.